floodmodeller_api/test/test_hydrographplus.py

```diff
@@ -0,0 +1,80 @@
+"""Tests to check the class HydrographPlus"""
+
+import os
+from pathlib import Path
+
+import pandas as pd
+import pytest
+
+from floodmodeller_api.hydrology_plus.hydrograph import HydrographPlusExport
+
+
+@pytest.fixture()
+def metada_csv():
+    """To extract the metadata from the csv to be compared with the class"""
+
+    return {
+        "Hydrograph Name": "Baseline unchecked",
+        "Hydrograph Description": "",
+        "Calculation Point": "CP_003",
+        "ReFH2 Name": "CP_003_ReFH2_1",
+        "Winfap Name": "CP_003_WINFAP_1",
+        "Urban/Rural": "Urban",
+        "Urban/Rural Comment": "",
+        "ReFH2 Comment": "",
+        "Winfap Comment": "",
+        "Winfap Distribution": "GEV",
+        "Winfap Distribution Comment": "",
+        "Use Climate Change Allowances": "True",
+        "Use Custom Scale Factors": "False",
+        "Created By": "KA007155",
+        "Created Date": "30/04/2024 09:42:23",
+        "Checksum": "ef77d9bd-2eb3-4689-a1e3-665d293db810",
+    }
+
+
+@pytest.fixture()
+def dataframe_csv():
+    """To extract the df with all the flows to be compared with the df of the class"""
+    test_workspace = Path(os.path.dirname(__file__), "test_data")
+
+    return pd.read_csv(test_workspace / "df_flows_hplus.csv", index_col=0)
+
+
+@pytest.fixture()
+def event_csv():
```

Comment:
for this fixture as well as `dataframe_csv` fixture, avoid using the `_csv` suffix as it does not match the datatypes. also it is useful to name with `expected_` prefix so it is more clear. e.g. `expected_dataframe` and `expected_series`